index.html

```diff
@@ -167,6 +167,22 @@ <h3 id="configuration">
         Which Debian mirror to use. By default, <tt>ftp.de.debian.org</tt> is used.
       </dd>
       <dt>
+        TRAVIS_EXTRA_REPOSITORY
+      </dt>
+      <dd>
+        If the package has some dependencies not (yet) found in Debian, you can
+        add your own repository to the build using this variable. The format is
+        the same as in <tt>/etc/apt/sources.list</tt>, for example
+        <tt>"https://my.repo.net/debian/ stretch main"</tt>.
+      </dd>
+      <dt>
+        TRAVIS_EXTRA_REPOSITORY_GPG_URL
+      </dt>
+      <dd>
+        From where to download the public key of the extra repository, so that
```

Comment:
"From where to download" → "Location of"

"extra repository" → "extra repository defined in TRAVIS_EXTRA_REPOSITORY"